src/cli/deploy.rs

```diff
@@ -13,6 +13,7 @@ use crate::backend::Signatures;
 use anyhow::{anyhow, bail, Context};
 use camino::Utf8PathBuf;
 use clap::Args;
+use drawbridge_client::API_VERSION;
```

Comment:
I don't think this belongs here, it's not clear what "API_VERSION" is just by itself.